.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+"Area-GitHub workflow":
+  - .github/**/*
```

Comment:
Do you really need the extra `/*`? I'd hope not (normally `**` means any string of characters including `/`)

Reply:
The docs: https://github.com/actions/labeler#basic-examples

Reply:
Huh. I'll probably have to ask them. In some cases it might make sense if there's an override, but I suspect that a bunch of their examples could be simplified.